### PR TITLE
@kanaabe => Yoast snippet properly reflects the title Artsy uses for Google

### PR DESCRIPTION
**Pull request by mdole**
Had initially set the title for yoast's snippet editor (which imitates the appearance of the page on a google SERP) to be the article title; Artsy uses the thumbnail title instead.

### CODE REVIEW

**Comment by mdole** [comment]
cc @kanaabe

File: client/apps/edit/components/layout/index.coffee [hunk 1]
```diff
@@ -188,9 +188,15 @@ module.exports = class EditLayout extends Backbone.View
...
+    @yoastTitle = ''
+    if @article.get('thumbnail_title')
+      @yoastTitle = @article.get('thumbnail_title')
+    else
+      @yoastTitle = @article.get('title')
+
     yoastView = new YoastView
       contentField: @fullText
-      title: @article.get('title')
+      title: @yoastTitle
```
Comment (kanaabe) [inline client/apps/edit/components/layout/index.coffee]: ooh we can make this a one-line PR, ready? 
```
title: @article.get('thumbnail_title') or @article.get('title') or ''
```
and then if possible, we should add a test! you should be able to see what gets passed into YoastView by doing something after [this line](https://github.com/artsy/positron/blob/master/client/apps/edit/components/layout/test/index.coffee#L152)
```
console.log @EditLayout.YoastView.args
```

**Comment by mdole** [comment]
@kanaabe the one-liner is so much more elegant, thanks. I wasn't able to figure out a test that worked--I think we had some problems with this when we were designing the original tests, as we ended up checking fullText instead of the yoast args. Thoughts?

**Comment by kanaabe** [comment]
Ah yeah, you're right. Merging for now and I'll try to iron out the YoastView args for tests sometime later. Probably will become important to test YoastView args eventually as this module get more feature-packed and aWeSoMe

**Pull request merged**